instrumentation/opentelemetry-instrumentation-dbapi/tests/test_dbapi_integration.py

```diff
@@ -592,6 +592,49 @@ def test_instrument_connection(self):
         connection2 = dbapi.instrument_connection(self.tracer, connection, "-")
         self.assertIs(connection2.__wrapped__, connection)
 
+    @mock.patch("opentelemetry.instrumentation.dbapi.DatabaseApiIntegration")
+    def test_instrument_connection_kwargs_defaults(self, mock_dbapiint):
+        connection = mock.Mock()
+        # Avoid get_attributes failing because can't concatenate mock
```

Comment:
Not sure what this is referring to specifically, but you could probably achieve with MagicMock
```py
In [4]: mock.MagicMock() + []
Out[4]: <MagicMock name='mock.__add__()' id='140567852922320'>
```

Reply:
I copied it from an existing test 😶 It seems not needed at all for the tests I've added, so removed in [ce86c1e](https://github.com/open-telemetry/opentelemetry-python-contrib/pull/3027/commits/ce86c1e3a1080ed574e136fe5cd525af072cc12b)